lib/devices/ios/safari.js

```diff
@@ -69,13 +68,45 @@ Safari.prototype.start = function (cb, onDie) {
 
   this._start(function (err) {
     if (err) return cb(err);
-    this.navToInitialWebview(cb);
+
+    if (this.args.udid) {
+      logger.debug("On Safari Launcher. Tapping button to launch Safari");
```

Comment:
would prefer to wrap all of this up in its own method (maybe `startSafariFromLauncher` or something)